.github/workflows/ci.yml

```diff
@@ -16,20 +16,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.4", "2.5", "2.6", "2.7"]
-        rails: ["5", "6"]
+        ruby: ["2.5", "2.6", "2.7", "3.0"]
+        rails: ["5", "6.0", "6.1"]
```

Comment:
Since the code was working with Rails 6.0 but not 6.1, we now make a distinction here.